apps/desktop/src/lib/stacks/stacksService.ts

```diff
@@ -0,0 +1,50 @@
+import { reduxApi } from '$lib/redux/api';
```

Comment:
Probably it is a better form to call this "stackService" instead of pluralizing. This, as well as all other ones where you append @mtsgrd